lua/orgmode/objects/calendar.lua

```diff
@@ -156,25 +158,31 @@ function Calendar:render()
     weekday_row = { 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat' }
   end
 
+  local date = self.date or Date.today()
```

Comment:
Can it happen that we don't have a `self.date` ?

Reply:
This is actually a form of defensive programming and came in as an attempt to fix the bug I initially observed. Removed.